### ISSUE TITLE
Clinical validation using MJFF Levodopa Response Study

### ISSUE THREAD

**Issue by angerhang** [OxWearables/ssl-wearables#12]
- [ ] Process the MJFF with labels in 10 seconds epoch.
* Expected data format can be found [here](https://zenodo.org/record/6574265#.YovCMi8w1qs).
* Label cleaning: merging walking classes that might be the same from an ACC perspective and annotate the merging process 
* Data quality check and create a PR to merge the data processing script into https://github.com/OxWearables/ssl-wearables/tree/main/data_parsing

Experiments 
Both the below experiments can be run using https://github.com/OxWearables/ssl-wearables/blob/main/downstream_task_evaluation.py

Example scripts to run for different modes. Ask Hang for model weights for different pre-trained models. 
```python 
python downstream_task_evaluation.py data=pamap_10s evaluation=hand_crafted
python downstream_task_evaluation.py gpu=1 data=pamap_10s evaluation=flip_net_random_mlp model=resnet
python downstream_task_evaluation.py gpu=0 data=pamap_10s evaluation=flip_net_ft_freeze model=resnet
python downstream_task_evaluation.py gpu=2 data=pamap_10s evaluation=flip_net_ft model=resnet
python downstream_task_evaluation.py gpu=3 data=pamap_10s evaluation=flip_net_100k_ft model=resnet
```
- [x] Add MJFF results to Table 2: RF+fine-tuning results 
- [x] Add MJFF results to Table 3: fine-tuning results 
- [ ] Add PCA MJFF results to Figure 2: feature visu. Ask Shing for code pointer 

The model weights for Table 2 can be downloaded from Github readme links. For Table 3, self-supervised ones, you can get from Github readme links too. Ask Hang for the supervised weights.

Writing 
- [x] Add MJFF descriptions to Table 4: dataset descriptions

**Comment by aidanacquah** [comment]
PR submitted for code to download data
